### PR TITLE
Merge pre-release branch of 1.10.0 back to main

### PR DESCRIPTION
**Pull request by alkwa-msft**
# What
Deleting extra changelogs, bumping version of communication-react, merging in changelog changes.

# Why
Part of our stable release process

### CODE REVIEW

**Review by palatter — APPROVED** [review]

**Review by mgamis-msft — APPROVED** [review]

File: packages/communication-react/CHANGELOG.stable.md [hunk 1]
```diff
@@ -1,9 +1,49 @@
 # Change Log - @azure/communication-react
 
-This log was last generated on Fri, 06 Oct 2023 19:59:45 GMT and should not be manually modified.
+This log was last generated on Mon, 06 Nov 2023 16:44:11 GMT and should not be manually modified.
 
 <!-- Start content -->
 
+## [1.10.0](https://github.com/azure/communication-ui-library/tree/@azure/communication-react_v1.10.0)
```
Comment (mgamis-msft) [inline packages/communication-react/CHANGELOG.stable.md:7]: Missed this in the grooming step. This should be '## [1.10.0](https://github.com/azure/communication-ui-library/tree/1.10.0)'

File: packages/communication-react/CHANGELOG.stable.md [hunk 2]
```diff
@@ -1,9 +1,49 @@
...
+
+Mon, 06 Nov 2023 16:44:11 GMT 
+[Compare changes](https://github.com/azure/communication-ui-library/compare/@azure/communication-react_v1.10.0-beta.1...@azure/communication-react_v1.10.0)
```
Comment (mgamis-msft) [inline packages/communication-react/CHANGELOG.stable.md:10]: We missed this in the grooming. This should be:
`[Compare changes](https://github.com/azure/communication-ui-library/compare/1.10.0-beta.1...1.10.0)`

File: packages/communication-react/CHANGELOG.stable.md [hunk 3]
```diff
@@ -1,9 +1,49 @@
...
+
+You can now add your own custom buttons to the CallComposite and CallwithChatComposite on web (on desktop and mobile web). Custom Button Injection enables developers to add their own custom buttons to the call control bar component. Developers have the ability to add buttons to different sections of the call control bar: primary, secondary, or overflow areas of the control bar. With our simplified button API, you can now create a more immersive calling experience that integrates with your applications.
```
Comment (mgamis-msft) [inline packages/communication-react/CHANGELOG.stable.md:12]: Should have a sub-heading here for precedence like:
`### Custom Button Injection - General Availability`

File: packages/communication-react/CHANGELOG.stable.md [hunk 4]
```diff
@@ -1,9 +1,49 @@
...
-This log was last generated on Fri, 06 Oct 2023 19:59:45 GMT and should not be manually modified.
+This log was last generated on Mon, 06 Nov 2023 16:44:11 GMT and should not be manually modified.
 
 <!-- Start content -->
 
+## [1.10.0](https://github.com/azure/communication-ui-library/tree/@azure/communication-react_v1.10.0)
+
+Mon, 06 Nov 2023 16:44:11 GMT 
+[Compare changes](https://github.com/azure/communication-ui-library/compare/@azure/communication-react_v1.10.0-beta.1...@azure/communication-react_v1.10.0)
+
```
Comment (mgamis-msft) [inline packages/communication-react/CHANGELOG.stable.md:11]: Should have a sub-heading here for precedence like:
`### Custom Button Injection - General Availability`

Comment (alkwa-msft) [reply packages/communication-react/CHANGELOG.stable.md:11]: changed. Will only affect what we merge back into main

File: packages/communication-react/CHANGELOG.stable.md (hunk 2 above)
Comment (alkwa-msft) [reply packages/communication-react/CHANGELOG.stable.md:10]: shouldn't we really be comparing against 1.9.0? @mgamis-msft

Comment (mgamis-msft) [reply packages/communication-react/CHANGELOG.stable.md:10]: Yeah that makes sense to me. We may need to change something in the beachball config to do this from now on. In the previous releases we have been comparing to the previous beta.

**Pull request merged**